.github/workflows/import_count.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         python ./scripts/count-trans-deps.py "Mathlib/" > head.json
 
         # Checkout the merge base
-        git checkout master # let's start easy! "$(git merge-base refs/remotes/origin/main ${{ github.sha }})"
+        git checkout master
```

Comment:
I'm not sure this is a good idea - if an import is removed in master while the pr is open, I suspect this will blame the open PR for "adding" it back

Reply:
Aaah, that's a good point.